templates/kubelet.service.erb

```diff
@@ -10,7 +10,13 @@ Requires=docker.service
 WorkingDirectory=<%= @kubelet_dir %>
 ExecStart=<%= scope['kubernetes::_dest_dir'] %>/kubelet \
   --v=<%= scope['kubernetes::log_level'] %> \
+<% if @_register_schedulable -%>
```

Comment:
This is a string and not a boolean, use the old code for versions before 1.6

Versions after 1.6.0 should register taints form a list like we do for node labels